### PR TITLE
PLANET-3759 - Convert Cookies block to Gutenberg

### PR DESCRIPTION
**Pull request by comzeradd**
This also contains some minor cleanup fixes:
* Added .git extension to submodule. It was removed at c207eb275813b7f48fc7d57d1fed74abeaef8946 but without it the submodule didn't work for me.
* Add `parserOptions` on eslint, so it doesn't complain for import/export.
* Added production mode to build command, since we practically only use this for production. We have the `start` command if we don't want minification.
* Removed unnecessary `.gitignore` files from a couple of folders.

### CODE REVIEW

**Comment by kirdia** [comment]
Looks good!
It needs this functionality though for the frontend.
[https://github.com/greenpeace/planet4-plugin-blocks/blob/develop/admin/js/blocks/CookiesBlock.js](url)

**Comment by comzeradd** [comment]
@kirdia PR amended.

File: classes/blocks/class-cookies.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+				$post = get_post( $post_id );
+			}
+		} else {
+			$post = get_queried_object();
+		}
+
+		// Enqueue js for the frontend.
+		if ( ! $this->is_rest_request() ) {
+			wp_enqueue_script( 'cookies', P4GBKS_PLUGIN_URL . 'public/js/cookies.js', [ 'jquery' ], '0.1', true );
+			wp_localize_script( 'cookies', 'cookies', $menu );
```
Comment (kirdia) [inline classes/blocks/class-cookies.php]: @comzeradd please remove this, is unneeded

**Comment by comzeradd** [comment]
@kirdia thanks! PR fixed

File: react-blocks/src/blocks/Cookies/CookiesBlock.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+          type: 'string',
+        },
+        all_cookies_name: {
+          type: 'string',
+        },
+        all_cookies_description: {
+          type: 'string',
+        },
+      },
+      edit: withSelect((select) => {
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Cookies/CookiesBlock.js]: As we are not using `withSelect` HOC here, we could remove it.

**Comment by comzeradd** [comment]
@sagarsdeshmukh PR updated

**Pull request merged**